cranelift/codegen/src/ir/instructions.rs

```diff
@@ -766,6 +769,47 @@ impl OperandConstraint {
                     .dynamic_to_vector()
                     .expect("invalid type for dynamic_to_vector"),
             ),
+            Narrower => {
+                let ctrl_type_bits = ctrl_type.log2_lane_bits();
+                let mut tys = ValueTypeSet::default();
+
+                // We're testing scalar values, only.
+                tys.lanes = BitSet::from_range(0, 1);
```

Comment:
It might be worth adding a `BitSet::singleton` function to make it a bit more obvious what's going on here.